bclientapi/bclientapi.go

```diff
@@ -44,13 +47,14 @@ func addFileToTransactionList(filename string, fileID string, item string, isNew
 	fileIDMutex.Unlock()
 }
 
-func New(server string, item string, fileroot string, chunkSize int) *itemAttributes {
+func New(server string, item string, fileroot string, chunkSize int, wait bool) *itemAttributes {
 
 	thisItem := new(itemAttributes)
 	thisItem.bendoServer = server
 	thisItem.item = item
 	thisItem.chunkSize = chunkSize
 	thisItem.fileroot = fileroot
+	thisItem.wait = wait
 
 	return thisItem
 }
```

Comment:
Just a comment, and doesn't need to be changed for this PR, but New() is getting a lot of parameters. Think about making all of these members in `itemAttributes` public, so the New() function is not needed? e.g.

```
type itemAttributes struct {
    BendoServer
    Item
    ChunkSize
    Fileroot
    Wait
}
```
And then to make a new one you just do
```
item := itemAttributes{
     BendoServer: server,
     ChunkSize: chunksize,
     ...
}
```

instead of passing all the parameters to New, which then turns around and does the exact same thing.